src/http/api/resources/config.js

```diff
@@ -1,10 +1,10 @@
 'use strict'
 
 const debug = require('debug')
-const get = require('lodash/get')
-const set = require('lodash/set')
-const log = debug('ipfs:http-api:config')
-log.error = debug('ipfs:http-api:config:error')
+const get = require('dlv')
+const set = require('just-safe-set')
+const log = debug('jsipfs:http-api:config')
+log.error = debug('jsipfs:http-api:config:error')
```

Comment:
These have been [standardized to all start with "ipfs:"](https://github.com/ipfs/js-ipfs/pull/1911).

```suggestion
log.error = debug('ipfs:http-api:config:error')
```